common/v2/components/AccountList.tsx

```diff
@@ -170,6 +214,34 @@ function buildAccountTable(
 
   return {
     head: deletable ? [...columns, translateRaw('ACCOUNT_LIST_DELETE')] : columns,
+    overlay:
+      overlayRows && overlayRows[0] !== undefined ? (
+        <TableOverlay>
```

Comment:
It seems that this code is duplicated in AccountList and AddressBook.
It seems like the right moment to create a `v2/component` for the overlay.

Reply:
Done!